requirements.txt

```
@@ -0,0 +1,3 @@
+pandas
+matplotlib
+colorama
```

```suggestion
colorama
numpy
pytz
```

Need to include also

```
openapi_client
openapi_genclient
```
?

---

If we add those two, pip install will fail. I don't think they are on pip.
I think they should rather be in the readme.

---

Вроде, пишут, что через pip можно с pypi. Только ссылку надо будет как-то указать.

https://github.com/Awethon/open-api-python-client

---

@Alexander-Serov ?

---

Не, это все фигня. Не сможем мы объяснить людям, которые захотят использовать, что нужно установить вот как-то сложно. Поэтому хорошо, если мы перейдем #22 в какой-то момент.

---

Окей, тогда оставим как ишшу